src/generated/cloud/aiplatform/v1/Cargo.toml

```diff
@@ -26,6 +26,74 @@ keywords.workspace     = true
 categories.workspace   = true
 rust-version.workspace = true
 
+[features]
+default = [
+    "dataset_service",
+    "deployment_resource_pool_service",
+    "endpoint_service",
+    "evaluation_service",
+    "feature_online_store_admin_service",
+    "feature_online_store_service",
+    "feature_registry_service",
+    "featurestore_online_serving_service",
+    "featurestore_service",
+    "gen_ai_cache_service",
+    "gen_ai_tuning_service",
+    "index_endpoint_service",
+    "index_service",
+    "job_service",
+    "llm_utility_service",
+    "match_service",
+    "metadata_service",
+    "migration_service",
+    "model_garden_service",
+    "model_service",
+    "notebook_service",
+    "persistent_resource_service",
+    "pipeline_service",
+    "prediction_service",
+    "reasoning_engine_execution_service",
+    "reasoning_engine_service",
+    "schedule_service",
+    "specialist_pool_service",
+    "tensorboard_service",
+    "vertex_rag_data_service",
+    "vertex_rag_service",
+    "vizier_service",
+]
+dataset_service = []
```

Comment:
forecasting: some, but not all of these will require gRPC. Will we micro-manage the features' dependencies?

I lean: nah.

Reply:
I think eventually: yes. But we can move forward without this.